bucket/winaero-tweaker.json

```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://winaero.com/comment.php?comment.news.1836",
+    "license": "Freeware",
+    "description": "Tweaks and hidden settings for power-users for Windows 7/8/8.1/10",
+    "version": "0.14",
+    "url": "https://winaero.com/winaerotweaker.zip",
+    "hash": "a73ddf39cc92213106573f974ac032b396b6ff793a48156484416a1f45aa4a98",
+    "installer": {
+        "file": "WinaeroTweaker-0.14.0.0-setup.exe",
+        "args": [
+            "/SP-",
+            "/VERYSILENT",
+            "/PORTABLE",
+            "/DIR=\"$original_dir\""
```

Comment:
Are we using `original_dir` as we’re running an installer extracted from the download?

Reply:
Felt more semantically correct to be explicit and install to the "0.14" directory as opposed to "current" directory.

Reply:
Changed to $dir per discussion below